### ISSUE TITLE
Don't work with unsaved contacts !

### ISSUE THREAD

**Issue by DarkLordZeus** [Amitpatil215/whatsapp_share#13]

**Comment by rahul-yr** [comment]
Any work around for this ?

**Comment by Amitpatil215** [comment]
**You can try using click to chat feature from WhatsApp**

## Create your own link
Use https://wa.me/<number> where the <number> is a full phone number in international format. Omit any zeroes, brackets, or dashes when adding the phone number in international format.
```Examples:
Use: https://wa.me/1XXXXXXXXXX
Don't use: https://wa.me/+001-(XXX)XXXXXXX
```

## Create your own link with a pre-filled message
The pre-filled message will automatically appear in the text field of a chat. Use https://wa.me/whatsappphonenumber?text=urlencodedtext where whatsappphonenumber is a full phone number in international format and urlencodedtext is the URL-encoded pre-filled message.

```Example: https://wa.me/1XXXXXXXXXX?text=I'm%20interested%20in%20your%20car%20for%20sale
To create a link with just a pre-filled message, use https://wa.me/?text=urlencodedtext
Example: https://wa.me/?text=I'm%20inquiring%20about%20the%20apartment%20listing
```
After clicking on the link, you’ll be shown a list of contacts you can send your message to.

Source : https://faq.whatsapp.com/5913398998672934/?helpref=uf_share

**Comment by rahul-yr** [comment]
> **You can try using click to chat feature from WhatsApp**
> 
> ## Create your own link
> Use https://wa.me/ where the is a full phone number in international format. Omit any zeroes, brackets, or dashes when adding the phone number in international format.
> 
> ```
> Use: https://wa.me/1XXXXXXXXXX
> Don't use: https://wa.me/+001-(XXX)XXXXXXX
> ```
> 
> ## Create your own link with a pre-filled message
> The pre-filled message will automatically appear in the text field of a chat. Use https://wa.me/whatsappphonenumber?text=urlencodedtext where whatsappphonenumber is a full phone number in international format and urlencodedtext is the URL-encoded pre-filled message.
> 
> ```
> To create a link with just a pre-filled message, use https://wa.me/?text=urlencodedtext
> Example: https://wa.me/?text=I'm%20inquiring%20about%20the%20apartment%20listing
> ```
> 
> After clicking on the link, you’ll be shown a list of contacts you can send your message to.
> 
> Source : https://faq.whatsapp.com/5913398998672934/?helpref=uf_share

Yes, that works irrespective of the saved contacts. But I can't attach files.

**Comment by Amitpatil215** [comment]
That's right @rahul-yr . For now I can't think of any workaround regarding `Sharing files to unsaved WhatsApp contacts`. Will keep you posted if I find anything.

**Comment by rahul-yr** [comment]
ok

**Comment by letsintegreat** [comment]
Have you guys figured out something yet, any other package maybe? I can't send files to unsaved numbers.

**Comment by Jenil-Infyom** [comment]
Add Below Flag into Native Android Code into shareFiles Functions
![Screenshot 2024-06-04 at 5 29 21 PM](https://github.com/Amitpatil215/whatsapp_share/assets/109334364/8e115113-7fc0-4111-bf04-c34f53c81b20)
